mypy.sh

```
@@ -8,32 +8,35 @@ RETVAL=0
 echo "running $SCRIPT_DIR/mypy.sh"
 
-cd $SCRIPT_DIR/engine
-echo "Checking $SCRIPT_DIR/engine"
-mypy --strict *.py
+export MYPYPATH="${SCRIPT_DIR}/stubs"
+MYPY=(mypy --strict --no-incremental --show-error-codes --pretty)
+
+echo cd "$SCRIPT_DIR/engine"
+cd "$SCRIPT_DIR/engine" || exit 1
+echo "${MYPY[@]}" *.py
+"${MYPY[@]}" *.py
```

Shellcheck (0.11.0)

[info] Use ./*glob* or -- *glob* so names with dashes won't become options.

(SC2035)